### ISSUE TITLE
build: build-osx.sh fails during the release build

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#14190]
The [release build](
https://teamcity.cockroachdb.com/viewLog.html?buildId=182663&buildTypeId=Cockroach_MergeToMaster) is failing during `build-osx.sh` in `make xgo-build`:

```
[23:25:07]+ docker tag cockroachdb/cockroach cockroachdb/cockroach:alpha-17638-g417a0da
[23:25:07]+ docker push cockroachdb/cockroach
[23:25:17]+ mv build/deploy/cockroach cockroach
[23:25:17]+ aws configure set region us-east-1
[23:25:17]+ build/build-osx.sh
[23:25:18]Makefile:330: *** mixed implicit and normal rules: deprecated syntax
[23:25:18]make: *** [.go-version] Error 1
[23:25:18]Process exited with code 2
```

Seems like this was caused by #14104, so assigning to @benesch to figure out what's up.

**Comment by benesch** [comment]
On it, thanks!

**Comment by a-robinson** [comment]
Fixed by #14191
